client.js

```diff
@@ -0,0 +1,62 @@
+const dotenv = require('dotenv');
+dotenv.config();
+
+const authServerURL = process.env.BASE_URL + ':' + process.env.AUTH_SERVER_PORT;
+const serviceURL = process.env.BASE_URL + ':' + process.env.SERVICE_PORT;
+
+const loginCredentials = {
+  login: 'user@email.com',
+  password: '12345678',
+};
+
+const getToken = async () => {
+  try {
+    console.log(new Date().toUTCString() + ' [Fetch token from Auth Server using credentials]');
+    const response = await fetch(`${authServerURL}/login`, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(loginCredentials),
+    });
+
+
+    if (!response.ok) {
+      throw new Error( new Date().toUTCString() + ` [Failed to fetch token - ${response.statusText}]`);
```

Comment:
```suggestion
      throw new Error(`Failed to fetch token - ${response.statusText`);
```
If you want to enrich the Error, use a custom Error subclass and add that as metadata, otherwise the error handling in the catch below will be very noisy.